src/main/java/frc/robot/RobotContainer.java

```diff
@@ -178,19 +181,19 @@ private void configureButtonBindings() {
       m_buttonBox.button2.whenPressed(new ConditionalCommand( 
           new ParallelCommandGroup(
               new ArmMotionMagic(m_arm, m_armLayupAngle.getValue()), 
-              new ShooterFlywheelRun(m_shooter, m_shooterLayupSpeed.getValue()),
-              new LimelightToggle(m_sensors, true)),
```

Comment:
I intentionally removed this command to turn the limelight on...we don't need it for the layup shot.